### PR TITLE
Added dtreeviz to visualize the DecisionTree

### PR DESCRIPTION
**Pull request by Hk669**
Successfully added the dtreeviz to visualize the Decision tree, and go through the code.
Imported the necessary libraries to perform the visualization.

### CODE REVIEW

**Comment by pplonski** [comment]
The issue wasn't in the test. The test is to check if the visualization is working. Did you manage to fix visualization in the code?

**Comment by Hk669** [comment]
yes, I did! added the code to visualize.

**Comment by pplonski** [comment]
The issue was that there are no decision tree plots in the report generated with AutoML. Did you fixed the issue with your changes?

**Comment by Hk669** [comment]
Yes! After running the AutoML which is fit on the training data, it's gonna save the Decision Tree visualization in .svg format in the 1_DecisionTree directory.

File: tests/tests_automl/test_explain_levels.py [hunk 1]
```diff
@@ -254,73 +257,88 @@ def test_explain_just_permutation_importance(self):
...
+        # - decision tree visualization
+        # - permutation importance
+        # - shap importance
+        # - shap dependence
+        # - shap decisions
+
+        # Check Decision Tree visualization
+        produced = False
+        for f in result_files:
+            if "tree.svg" in f:
```
Comment (pplonski) [inline tests/tests_automl/test_explain_levels.py:295]: here is a check if there is `tree.svg` file in the report

File: tests/tests_automl/test_explain_levels.py [hunk 2]
```diff
@@ -254,73 +257,88 @@ def test_explain_just_permutation_importance(self):
...
+        self.assertTrue(produced)
+        # Check shap decisions
+        produced = False
+        for f in result_files:
+            if "decisions.png" in f:
+                produced = True
+                break
+        self.assertTrue(produced)
+
+        # dtreeviz visualization of DecisionTree
```
Comment (pplonski) [inline tests/tests_automl/test_explain_levels.py:331]: no need to create a tree visualization in the test, below code is not needed

**Review by pplonski — COMMENTED** [review]
is this test working after uncommenting?

**Comment by Hk669** [comment]
Nope, the unittest is failed. There is an issue with the dtreeviz module.

**Comment by Hk669** [comment]
yep, it's working. The issue is the visualization is not showing up in the directory. If this is the problem we need to add the code for visualization, am I right? if not, please guide me.

**Comment by pplonski** [comment]
The code for tree visualization in AutoML reports is ready. It is using `dtreeviz`. After some time, the visualization images disappeared from reports. Because of this test was failing. I disabled it.

To fix this issue, you need to:
- find the reason why plots with decision trees are not created any more,
- fix the issue,
- re-enable the unit test.

**Comment by pplonski** [comment]
I'm closing the PR because it is not solving the problem with decision tree visualisation.

**Pull request closed without merging**